test/services/MeasureService.test.ts

```diff
@@ -390,26 +409,46 @@ describe('MeasureService', () => {
           resourceType: 'Parameters',
           parameter: [
             { name: 'id', valueString: 'testWithRootLibAndDeps' },
-            { name: 'periodStart', valueDate: '2022-01-01' },
-            { name: 'periodEnd', valueDate: '2022-12-31' }
+            { name: 'periodStart', valueDate: '2021-01-01' },
+            { name: 'periodEnd', valueDate: '2021-12-31' }
           ]
         })
         .set('content-type', 'application/fhir+json')
         .expect(200)
         .then(response => {
           expect(response.body.resourceType).toEqual('Library');
+          expect(response.body.type.coding[0].code).toEqual('module-definition');
           expect(response.body.dataRequirement).toHaveLength(0);
+          expect(calc).toBeCalledWith(
+            expect.objectContaining({
+              resourceType: 'Bundle'
+            }),
+            expect.objectContaining({
+              measurementPeriodStart: '2021-01-01',
+              measurementPeriodEnd: '2021-12-31'
+            })
+          );
         });
     });
 
     it('returns 200 with query params', async () => {
       await supertest(server.app)
-        .get('/4_0_1/Measure/$data-requirements?id=testWithRootLibAndDeps&periodStart=2022-01-01&periodEnd=2022-12-31')
-        .set('content-type', 'application/fhir+json')
+        .get('/4_0_1/Measure/$data-requirements')
+        .query({ id: 'testWithRootLibAndDeps', periodStart: '2021-01-01', periodEnd: '2021-12-31' })
```

Comment:
Right now, we test periodStart and periodEnd when they are passed through
1. Parameter
2. Query
Would it be beneficial to add one more that passes the periodStart and periodEnd through the url like this test used to? I think this was a good change here but maybe we can have both? Or is that redundant? If we do add it, probably would be good to use 2021 like the others :)

Reply:
I believe that this replacement is equivalent to putting the query parameters directly into the url. I made this change to standardize with other tests in this file that use `.query` instead.

Reply:
Sounds good!